NASA-AMMOS/aerie-ui
# Toggle view panels via new menu buttons

This resolves #345.

To test:

1. Create a plan and navigate to it
2. Hover over the View menu at the top right
3. Click one of the buttons to toggle on/off panels
4. Verify that the intended panel is shown/hidden
5. Verify that panels are resizable
6. Save the view
7. Verify that reloading the view loads the view state correctly
